USAGE.md

```diff
@@ -109,20 +111,21 @@ The Manubot workflow requires the bibliographic details for references (the set
 The Manubot attempts to automatically retrieve metadata and generate valid citeproc JSON for references, which is exported to `output/references.json`.
 However, in some cases the Manubot fails to retrieve metadata or generates incorrect or incomplete citeproc metadata.
 Errors are most common for `url` references.
-For these references, you can manually specify the correct citeproc in [`content/manual-references.json`](content/manual-references.json), which will override the automatically generated reference data.
-To do so, create a new citeproc record that contains the field `"standard_citation"` with the appropriate reference identifier as its value.
+For these references, you can manually specify the correct CSL Data in [`content/manual-references.json`](content/manual-references.json), which will override the automatically generated reference data.
+To do so, create a new CSL JSON record that contains the field `"standard_citation"` with the appropriate reference identifier as its value.
 The identifier can be obtained from the `standard_citation` column of `citations.tsv`, which is located in the `output` branch or in the `output` subdirectory of local builds.
 As an example, `manual-references.json` contains:
 
 ```json
 "standard_citation": "url:https://github.com/greenelab/manubot-rootstock"
 ```
 
+The metadata for `raw` citations must be provided in `manual-references.json` or an error will occur.
```

Comment:
It would be very helpful to add an example raw citation in `manual-references.json` and provide the JSON snippet like you do for the example above.

Reply:
Does https://github.com/greenelab/manubot-rootstock/pull/141/commits/2ac833d0d537fdcf93343315681d25c8e49161f1 satisfy? I didn't add to `manual-references.json`, because I want to keep that file minimal (as every new instance inherits it, and conflicts from rootstock updates to it can be annoying to resolve). Instead, I added a bit more in the USAGE instructions